packages/server/postgres/migrations/1676299334703_addSeasonalTemplates.ts

```diff
@@ -0,0 +1,211 @@
+    templateId: makeId(templateNames[0], 'template'),
+    sortOrder: 2
+  },
+  {
+    question: 'Sweets',
+    description: 'Who deserves something sweet, kudos, or a word of thanks?',
+    templateId: makeId(templateNames[0], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Easter Eggs',
+    description:
+      'What was hidden from sight or caught you by surprise? What did you learn or find out?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Seeds',
+    description:
+      'What new processes would we like to cultivate? Is there something that faded away that we should bring back?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Hope',
+    description: 'What did you hope for? What are you hoping for next?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 2
+  },
+  {
+    question: 'Chocolate',
+    description: 'Who deserves some kudos or a word of thanks?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Wreaths',
+    description: 'What processes and practices are binding us together and making us stronger?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Schnapps',
+    description: 'What’s been hard to swallow, admit, or realize about our work?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Longest Day',
+    description: 'What feels like it took too long or was never-ending?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 2
+  },
+  {
+    question: 'Strawberry Cake',
+    description: 'Who deserves some kudos or a word of thanks?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Hongbao 🧧',
+    description: 'Who deserves a red envelope with kudos and some words of appreciation?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Couplets 🎊 ',
+    description: 'What important messages should we remember for guidance?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Firecracker 🧨',
+    description: 'What caught us by surprise or gave us a fright?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 2
+  }
+]
+
+const templates = templateNames.map((templateName) => makeTemplate(templateName))
+const reflectPrompts = promptsInfo.map((promptInfo, idx) => makePrompt(promptInfo, idx))
+
+const connectRethinkDB = async () => {
+  const {hostname: host, port, pathname} = new URL(process.env.RETHINKDB_URL!)
+  await r.connectPool({
+    host,
+    port: parseInt(port, 10),
+    db: pathname.split('/')[1]
+  })
+  return r as any as ParabolR
+}
+
+export const up = async function () {
+  try {
+    await connectRethinkDB()
```

Comment:
I see that everywhere else in the end of the function we call the `await r.getPoolMaster()?.drain()`
Do we need this here too or not necessary?